components/function-controller/cmd/webhook/main.go

```diff
@@ -26,11 +26,12 @@ import (
 )
 
 type config struct {
-	SystemNamespace    string `envconfig:"default=kyma-system"`
-	WebhookServiceName string `envconfig:"default=serverless-webhook"`
-	WebhookSecretName  string `envconfig:"default=serverless-webhook"`
-	WebhookPort        int    `envconfig:"default=8443"`
-	WebhookConfigPath  string `envconfig:"default=/appdata/config.yaml"`
+	SystemNamespace     string               `envconfig:"default=kyma-system"`
```

Comment:
I can move it all to package `webhook` and init config with prefix `WEBHOOK`

Reply:
That sounds like a great idea :+1: